core/store/models/job_run.go

```diff
@@ -283,7 +283,7 @@ func (tr *TaskRun) ApplyOutput(result RunOutput) {
 // RunResult keeps track of the outcome of a TaskRun or JobRun. It stores the
 // Data and ErrorMessage.
 type RunResult struct {
-	ID           uint        `json:"-" gorm:"primary_key;auto_increment"`
+	ID           uint32      `json:"-" gorm:"primary_key;auto_increment"`
```

Comment:
Bigserial is 32 bits, so if you want to specify width, I'd specify at least 64.

Reply:
This is a regular `SERIAL` (integer) postgres type which is 32 bits... so uint32 is enough, surely?